### DISCUSSION TITLE
Adding Storybook to an existing project that requires using --legacy-peer-deps

### DISCUSSION THREAD

**Discussion by Reikon95** [storybookjs/storybook#25227] [Help] [answered]
### Summary

I'm currently trying to add storybook to an existing react project that due to unrelated issues, requires the --legacy-peer-deps flag when installing deps.

Because setup is an npx process, it won't accept the flag when I run npx storybook@latest init. What's the best way around this? Thank you.

**Comment by Marklb** [comment, marked as answer]
I don't know about a best way, but you could try creating an `.npmrc` file at the root of your repo, containing `legacy-peer-deps=true`, which I don't think will get ignored by how Storybook runs the commands.

**Reply by Reikon95** [reply]
That worked great, thank you :)
